core/src/epicli/data/common/defaults/configuration/postgresql.yml

```diff
@@ -105,19 +105,10 @@ specification:
         # By default 'logging_collector' is enabled and there is up to 7 files named 'daily' (e.g. postgresql-Wed.log)
         # and thus they can be overwritten by built-in log facility to provide rotation.
     # To have similar configuration for both distros (with logrotate), 'log_filename' parameter is modified.
-    config: |-
-      /var/log/postgresql/postgresql*.log {
-          maxsize 10M
-          daily
-          rotate 6
-          copytruncate
-      # delaycompress is for Filebeat
-          delaycompress
-          compress
-          notifempty
-          missingok
-          su root root
-          nomail
-      # to have multiple unique filenames per day when dateext option is set
-          dateformat -%Y%m%dH%H
-      }
+    pgbouncer:
+      period: weekly
+      number: 5
+    postgresql:
+      period: daily
+      number: 6
+      maxsize: 10
```

Comment:
`maxsize: 10` means rotate after 10 bytes so you would have only last 70 bytes (7 logs x 10B)